examples/networking-routerconfig-playbook/site.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Router Configurations
+  hosts: routers
+  gather_facts: no
+  connection: local
+  vars:
+    ansible_network_os: ios
+    dns_servers:
+      - 8.8.8.8
+      - 8.8.4.4
+    #Variables can be manually set like this:
+    #host1_private_ip: "172.18.2.8"
+    #control_private_ip: "172.17.1.157"
+    #or reference dynamically variables tied to the host directly
+    #in this case, its grabbing this from the inventory under lab_inventory:
+    host1_private_ip: "{{hostvars['host1']['private_ip']}}"
+    control_private_ip: "{{hostvars['ansible']['private_ip']}}"
+  tasks:
+    ##Configuration for R1
```

Comment:
From what I am seeing this one play should be two separate plays. One that RT2 `net_interface` and `ios_config` and another that does `net_static_route` and `net_system` for all routers. The hardcoding of host names and conditionals and overriding the host flow control is something we should be advocating.